serverless/knative_eventing/serverless-kn-trigger.adoc

```diff
@@ -56,18 +30,20 @@ spec:
 <2> The name of the broker where events will be filtered from. If the broker is not specified, the trigger will revert to using the `default` broker.
 <3> The name of the service that will consumer filtered events.
 
-include::modules/kn-trigger-list.adoc[leveloffset=+1]
-include::modules/kn-trigger-describe.adoc[leveloffset=+1]
-
-[id="delete-kn-trigger"]
-== Deleting a trigger using `kn`
+The Knative CLI provides a set of `kn trigger` commands that can be used to create and manage triggers.
 
-.Procedure
+[id="serverless-kn-trigger-prerequisites"]
+== Prerequisites
 
-To delete a trigger, enter the following command:
-----
-$ kn trigger delete <TRIGGER-NAME>
-----
+* Knative Eventing and the `kn` CLI are installed on your cluster.
+* An available broker exists; either the `default` broker, or one that you have created.
++
+You can create the `default` broker either by following the instructions on xref:../serverless-using-brokers.adoc#serverless-using-brokers[Using brokers with Knative Eventing], or by using the `--inject-broker` flag while creating a trigger. Use of this flag is described in the procedure below.
```

Comment:
This xref doesn't work when I try it. 

It looks like you back up a directory, but then don't come back down. And you should go the whole way up (so back up two directories) and then come back down into serverless/knative_eventing